TI_Practice1/Circle.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Practice1
+{
+    class Circle
+    {
+        public Circle(double val)
+        {
+            radius = val;
+        }
+
+        readonly double radius;
```

Comment:
1. Could you please name the class field starting with "_" - _radius?
2. Could you also please add a property for the _radius field?
3. In addition, the class components should be arranged in the following order:
- fields
- properties
- methods

Reply:
1. DONE. But can you please explain why should I do it (probably during the next practice or lecture)? As I understand we should use the underscore in order not to create new variables' names when this. is not used. Isn't it?
2. DONE
3. DONE
4. DONE. As I understood readonly is an access modifier too but it can be used with properties?

Reply:
sure, let's discuss it if you still have questions